src/main/sensors/gyro.c

```diff
@@ -244,9 +244,10 @@ static void initGyroFilter(filterApplyFnPtr *applyFn, filter_t state[], uint16_t
 {
     *applyFn = nullFilterApply;
     if (cutoff > 0) {
-        *applyFn = (filterApplyFnPtr)pt1FilterApply;
```

Comment:
This function should not be modified in this way, because it changes how both gyro LPF work. By doing so you just enabled double LULU on gyro data

The distinction should happen inside `static void gyroInitFilters(void)` by adding a new setting to run lulu on gyro